.github/dependabot.yml

```diff
@@ -19,12 +19,12 @@ updates:
     schedule:
       interval: "weekly"
     groups:
-      alldependencies:
+      github-action-dependencies:
         patterns:
           - "*"
         update-types:
           - "minor"
           - "patch"
     reviewers:
       - "@MikeNeilson"
```

Comment:
I presume making a CDA-developers GitHub Team requires USACE org creation?

Reply:
I think so. I know we have one, but I recall having issues getting contractors into it previously. But that may have been before I created this. I will investigate and perhaps change in a separate PR.